gips/data/prism/prism.py

```diff
@@ -50,7 +50,6 @@ class prismRepository(Repository):
     name = 'PRISM'
     description = 'PRISM Gridded Climate Data'
     _datedir = '%Y%m%d'
-    _defaultresolution = [4000.0, 4000.0]
```

Comment:
This line was wrong, but should be moved to `prismAsset`, because it is used in core:
```
src/gips/gips/data/core.py:    _defaultresolution = [30.0, 30.0]
src/gips/gips/data/core.py:            res = self.Asset._defaultresolution
```